File: src/main/java/com/gdgswu/planeat/domain/home/HomeResponse.java
package com.gdgswu.planeat.domain.home;

import com.gdgswu.planeat.domain.history.HistoryResponse;
import lombok.Builder;
import lombok.Getter;

import java.util.List;

@Getter
@Builder
public class HomeResponse {
    private String userName;

    private Double todayTotalCal;
    private Double todayTotalCarb;
    private Double todayTotalProtein;
    private Double todayTotalFat;

    private List<HistoryResponse> todayHistories;
}

File: frontend/app/src/main/java/com/example/planeat/data/remote/dto/HomeResponse.kt
package com.example.planeat.data.remote.dto

data class HomeResponse(
    val success: Boolean,
    val data: HomeData?
)

data class HomeData(
    val userName: String,
    val todayTotalCal: Double,
    val todayTotalCarb: Double,
    val todayTotalProtein: Double,
    val todayTotalFat: Double
)